File: android/app/src/main/java/com/example/test_face_verification/fv/data/FVData.java
package com.example.test_face_verification.fv.data;

import java.io.Serializable;

public class FVData implements Serializable {
    private String id;
    private String employeeId;
    private String imageUrl;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getEmployeeId() {
        return employeeId;
    }

    public void setEmployeeId(String employeeId) {
        this.employeeId = employeeId;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }
}

File: lib/fv_data.dart
class FVData {
  String employeeId;
  String imageUrl;

  FVData(this.employeeId, this.imageUrl);

  Map<String, dynamic> toMap() {
    return {
      'employeeId': employeeId,
      'imageUrl': imageUrl,
    };
  }
}
